prow/cmd/deck/static/prow/prow.ts

```diff
@@ -645,23 +645,17 @@ function redraw(fz: FuzzySearch): void {
         max = 2 * 3600;
     }
     drawJobHistogram(totalJob, jobHistogram, now - (12 * 3600), now, max);
-    if (rerunStatus === "error") {
+    if (rerunStatus === "gh_redirect") {
         modal.style.display = "block";
-        rerunCommand.innerHTML = "You don't have permission to rerun that job";
-    } else if (rerunStatus === "success") {
-        modal.style.display = "block";
-        rerunCommand.innerHTML = "Job successfully triggered. Wait 30 seconds and refresh the page for the job to show up";
-    } else if (rerunStatus != null) {
-        modal.style.display = "block";
-        rerunCommand.innerHTML = "Nice try! The direct rerun feature hasn't been implemented yet, so that button does nothing.";
+        rerunCommand.innerHTML = "Rerunning that job requires GitHub login. Now that you're logged in, try again";
```

Comment:
Can we still direct things to Cole as a fallback?

Reply:
Redirecting this to @cjwagner